mkdocs.yml

```diff
@@ -1,42 +1,85 @@
 site_name: NeonAI® Documentation
-site_url: https://neongeckocom.github.io/neon-docs
+site_url: https://mikejgray.github.io/neon-docs
```

Comment:
I'm confused by this change, it's probably just that I don't yet understand this process well enough :-) but I marked it just in case I'm wrong

Reply:
This should be changed back before merging this PR

Reply:
Change still needed :-)